src/Libraries/RevitNodes/Elements/DividedPath.cs

```diff
@@ -239,7 +244,12 @@ public static DividedPath ByCurveAndDivisions(Autodesk.DesignScript.Geometry.Cur
 
             return new DividedPath(new[] { ElementCurveReference.TryGetCurveReference(curve) }, divisions);
         }
-
+        /// <summary>
+        /// Creates a revit divided path on curves given curves and number of division
```

Comment:
Creates a Revit divided path on the given collection of curves with specified amount of division